packages/patternfly-4/react-core/src/tsconfig.dts.json

```diff
@@ -8,16 +8,28 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "skipLibCheck": true,
-    "sourceMap": false, // allow sourcemap support
-    "lib": ["es2017", "dom"],
+    "sourceMap": false,
```

Comment:
Comments are not allowed in json files

Reply:
How did this work before?

Reply:
Gatsby is now reading json files in and parsing them so the newly added plugin doesn't like comments in json

Reply:
https://stackoverflow.com/questions/244777/can-comments-be-used-in-json